### PR TITLE
Fix eslint prettier config.

### PR DESCRIPTION
**Pull request by Jeanmsilva89**
# Summary

Addressing issue #54 by fixing the eslint config to use prettier. 

PS. I actually added the common eslint and prettier config from https://github.com/react-native-community/.github/blob/master/Guidelines/Repository%20Setup.md, however, it seemed like it was a bigger lift as there were quite a few places where the code would have to be updated. If the intention is to implement the common config on this repo, I'm happy to to update this PR to include it. 

## Checklist

- [x] I tested this thoroughly
- [ ] I added the documentation in `README.md` (if needed)

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "extends": "react-app",
   "plugins": ["prettier"],
   "rules": {
-    "jsx-a11y/accessible-emoji": 0
+    "jsx-a11y/accessible-emoji": 0,
```
Comment (lucasbento) [inline .eslintrc]: I would rather not turn this off but fix the place that throws this warning, would you mind doing that?

File: src/__fixtures__/releases/0.59.js [hunk 2]
```diff
@@ -1,11 +1,9 @@
 export default {
   usefulContent: {
-    description:
-      'This is the very nice 0.59 release!',
+    description: 'This is the very nice 0.59 release!',
     links: [
       {
-        title:
-          'This is a very cool link to a blog post about 0.59',
+        title: 'This is a very cool link to a blog post about 0.59',
```
Comment (lucasbento) [inline src/__fixtures__/releases/0.59.js]: I wonder why some trailing commas have been removed but this one did not, do you know the reason?

Comment (Jeanmsilva89) [reply src/__fixtures__/releases/0.59.js]: @lucasbento It's not a trailing comma, the url prop comes right after it.. :) took me a little while to see it.

Comment (lucasbento) [reply src/__fixtures__/releases/0.59.js]: That's true, so sorry I missed it.

**Comment by kelset** [comment]
👏👏👏👏 thanks @Jeanmsilva89 !

**Pull request merged**